### PR TITLE
Cb 14

### PR DESCRIPTION
**Pull request by hamzaakyildiz**
I refactor the unittest and API's
Fixing a windows run command in README.MD
Swagger extension

### CODE REVIEW

**Review by asi345 — DISMISSED** [review]
Required changed are done as intended.

**Review by mrabiabrn — DISMISSED** [review]

**Review by asi345 — APPROVED** [review]

File: practice-app/apiDocs/home/homePage.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+Get recent posts of the followings of the given user
+This endpoints It takes a path parameter
```
Comment (CodeBlo) [inline practice-app/apiDocs/home/homePage.yml:2]: It seems this line didn't get edited. May be a little fix up?

File: practice-app/test/test_home.py [hunk 2]
```diff
@@ -1,43 +1,49 @@
...
-import requests
+import datetime
+from api.home.home import *
+from werkzeug.exceptions import HTTPException
+from unittest.mock import patch
+import mock
+
+
+
+class MyTestCase(unittest.TestCase):
```
Comment (CodeBlo) [inline practice-app/test/test_home.py:10]: MyTestCase doesn't feel like a good test class name. Might want to change it

**Review by CodeBlo — COMMENTED** [review]
There might be some changes needed for homePage.yml and test_home.py

File: practice-app/apiDocs/home/homePage.yml (hunk 1 above)
Comment (hamzaakyildiz) [reply practice-app/apiDocs/home/homePage.yml]: You are right. Fixed

File: practice-app/test/test_home.py (hunk 2 above)
Comment (hamzaakyildiz) [reply practice-app/test/test_home.py]: Fixed

**Review by asi345 — APPROVED** [review]
Works fine, improvements are done as needed, I could not see any problems overall.

**Review by mrabiabrn — APPROVED** [review]
I could not see any problem, it should work properly.

**Review by kocasariumut — APPROVED** [review]
works as intended

**Comment by hamzaakyildiz** [comment]
The review is done. The feedbacks are considered and  the issues addressed accordingly.

**Pull request merged**